decidim-conferences/app/controllers/decidim/conferences/conference_registrations_controller.rb

```diff
@@ -71,7 +71,7 @@ def ensure_signed_in
       end
 
       def conference
-        @conference ||= Conference.find_by(slug: params[:conference_slug])
+        @conference ||= Conference.where(organization: current_organization).find_by(slug: params[:conference_slug])
```

Comment:
I think we could move the condition to the `find_by`:
```suggestion
        @conference ||= Conference.find_by(slug: params[:conference_slug], organization: current_organization)
```